core/src/main/java/brooklyn/entity/rebind/persister/AbstractBrooklynMementoPersister.java

```diff
@@ -110,6 +115,11 @@ public void delta(Delta delta, PersistenceExceptionHandler exceptionHanlder) {
         memento.updateEnricherMementos(delta.enrichers());
         memento.updateCatalogItemMementos(delta.catalogItems());
     }
+
+    @Override
+    public void queueDelta(Delta delta) {
+        log.warn("Legacy persister ignoring queued delta: "+delta);
```

Comment:
Perhaps `throw new IllegalStateException("Not supported; use "+BrooklynMementoPersisterToObjectStore.class)`.
With this code, we'll log and ignore the requested delta, so the delta will never be applied. Better to fail than to do that, I think (unless the delta really is empty!)